includes/commands/class-mu-migration-users.php

```diff
@@ -88,11 +89,10 @@ public function update_passwords( $args = array(), $assoc_args = array() ) {
 	}
 
 	/**
-	 * Based on retrieve_passwords
+	 * Handles sending password retrieval email to user (based on retrieve_password).
 	 *
 	 * @param $user_data
```

Comment:
Actually this `users` command should be its own WP-CLI package, it doesn't make much sense to have it here, I have plans to officially deprecated this command and either create a new package or recommend another one.